P/PARMETIS/build_tarballs.jl

```diff
@@ -85,7 +84,7 @@ augment_platform_block = """
 """
 
 platforms = supported_platforms()
-platforms, platform_dependencies = MPI.augment_platforms(platforms; MPItrampoline_compat="5.2.1")
+platforms, platform_dependencies = MPI.augment_platforms(platforms; MPItrampoline_compat="5.3.1", OpenMPI_compat="4.1.6, 5")
```

Comment:
Uhm, if you aren't changing version number of the package you can't change an existing compat bound
```suggestion
platforms, platform_dependencies = MPI.augment_platforms(platforms; MPItrampoline_compat="5.2.1", OpenMPI_compat="4.1.6, 5")
```

Reply:
Thanks Mosè!
Yes, I just wanted to recompile PARMETIS without the symbolic link on Windows so I would like to not bump the version number if it's not needed.